geoips/commandline/list_available_modules.py

```diff
@@ -20,10 +20,12 @@
 from importlib import import_module
 import traceback
 import warnings
+import logging
 
 # Always actually raise DeprecationWarnings
 # Note this SO answer https://stackoverflow.com/a/20960427
 warnings.simplefilter("always", DeprecationWarning)
+LOG = logging.getLogger(__name__)
```

Comment:
This likely needs the same treatment as create_sector_image, doesn't it? It should call setup_logging?

Reply:
Done.